pkg/docker/config/config_test.go

```diff
@@ -216,10 +239,26 @@ func TestGetAuth(t *testing.T) {
 				if tc.sys != nil {
 					sys = tc.sys
 				}
-				username, password, err := GetAuthentication(sys, tc.hostname)
+				auth, err := GetCredentials(sys, tc.hostname)
 				assert.Equal(t, tc.expectedError, err)
-				assert.Equal(t, tc.expectedUsername, username)
-				assert.Equal(t, tc.expectedPassword, password)
+				assert.Equal(t, tc.expected.Username, auth.Username)
+				assert.Equal(t, tc.expected.Password, auth.Password)
+				assert.Equal(t, tc.expected.IdentityToken, auth.IdentityToken)
```

Comment:
Non-blocking: Doesn’t `assert.Equal(t, tc.expected, auth)` work? That was the point of using `expected` as a single unit.